form.py

```diff
@@ -0,0 +1,59 @@
+from flask_wtf import Form
+from wtforms import StringField, BooleanField, SelectField, TextAreaField, RadioField
+from flask_wtf.file import FileField
+from wtforms.validators import DataRequired, Email, URL
+from werkzeug import secure_filename
+
+class ApplicationForm(Form):
+    name = StringField('Name', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(),Email()])
+    phone = StringField('Phone')
+    city = StringField('City')
+    state = StringField('State')
+    zipcode = StringField('Zip Code')
+    citizenship = BooleanField('Verify you are a U.S. Citizen or Green Card holder')
```

Comment:
Would be good to add the help text somewhere: "For some positions, due to legislation or agency policy, U.S. citizenship is required. 

"Note: Any individual who is not a Federal employee or GSA contractor and who is legally authorized to work in the U.S. is eligible to apply."